scripts/components/HUD/Buttons/AudioButton.js

```diff
@@ -201,7 +220,7 @@ export default class AudioButton extends React.Component {
    * @param {function} onStop Callback
```

Comment:
Add JSDoc for the new parameter

```suggestion
   * @param {Function} onEnd Callback, run when the track ends
   * @param {Function} onStop Callback, run when the player is paused
```